ElrondNetwork/elrond-go
# export kibana url to external config

Exported Kibana's URL to `external.toml`. Useful when running a non-managed Elastic Search instance.

Examples:
`    KibanaURL         = "http://localhost:9200/_plugin/kibana"` when elastic is installed as managed
`    KibanaURL         = "http://localhost:5601"` when install separately

Testing procedure:
- run a testnet with kibana and elastic on
- without changing kibana settings and only update the `external.toml` setting for kibana path, everything should run normally. (check logs for kibana errors)
